src/wallet/db.cpp

```diff
@@ -554,7 +554,6 @@ bool CDB::Rewrite(CWalletDBWrapper& dbw, const char* pszSkip)
         }
         MilliSleep(100);
     }
-    return false;
```

Comment:
Yes the compiler should warn. For me this is too long for not to have a clear return value. But consider it a nit.